Source/KISAddonPickup.cs

```diff
@@ -719,6 +721,24 @@ public enum PickupMode {
     KISAddonCursor.StopPartDetection();
   }
 
+  bool TryPutDraggedPartIntoInventory() {
+    // While dragging a part KISAddonCursor and KISAddonPointer are not running, so neither will work here.
+    // TODO: Would be nice to have KISAddonCursor to provide better feedback. Or maybe create something like KISDragCursor?
+    var eva = Mouse.HoveredPart?.GetComponent<KerbalEVA>();
```

Comment:
Could you, please, stick to C# 5.0 syntax? C# 6.0 has some good things (like this ?. operator), but my building system is C# 5.0. One day I'll upgrade, but it won't happen today.

Reply:
Sure, no problem.